fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -507,7 +508,7 @@ public class SessionVariable implements Serializable, Writable, Cloneable {
 
     // if true, need report to coordinator when plan fragment execute successfully.
     @VariableMgr.VarAttr(name = ENABLE_PROFILE, alias = IS_REPORT_SUCCESS)
-    private boolean enableProfile = false;
+    private boolean enableProfile = Config.profile_enable;
```

Comment:
The session variables is persistent,  so I think if you change the Config.profile_enable, the enableProfile won't be changed?

Reply:
for global session variable this wont't be changed